src/lib/api/basyx-v3/api.ts

```diff
@@ -236,6 +270,52 @@ export const AssetAdministrationShellRepositoryApiFp = function (configuration?:
             };
         },
 
+        /**
+         * @summary Uploads a thumbnail to the specified Asset Administration Shell (AAS).
+         * @param {string} aasId - The unique identifier of the Asset Administration Shell.
+         * @param {Blob} image - The image file to be uploaded as the thumbnail.
+         * @param fileName - Name of the image file to be uploaded.
+         * @param {object} [options] - Optional. Override HTTP request options.
+         * @throws {RequiredError}
+         */
+        putThumbnailToShell(
```

Comment:
I believe this method is autogenerated, but I think it's kind of confusing that the method doesn't actually put the thubmanail to a show, but returns a callback function that does so. I think the method was easier to use and understand if you just directly perform the put inside the method. Or maybe there is a good reason for why it's like this that I don't understand :/.

Reply:
allright!